ee/api/ee_event_definition.py

```diff
@@ -30,13 +35,39 @@ class Meta:
             "volume_30_day",
             "query_usage_30_day",
             "last_seen_at",
+            "verified_at",
         ]
 
     def update(self, event_definition: EnterpriseEventDefinition, validated_data):
         validated_data["updated_by"] = self.context["request"].user
+        now = timezone.now()
+
+        if "verified" in validated_data and validated_data["verified"] and not event_definition.verified:
+            # Verify event only if previously unverified
+            validated_data["verified_by"] = self.context["request"].user
+            validated_data["verified_at"] = now
+            validated_data["verified"] = True
+
+        elif "verified" in validated_data and not validated_data["verified"]:
+            # Unverifying event nullifies verified properties
+            validated_data["verified_by"] = None
+            validated_data["verified_at"] = None
+            validated_data["verified"] = False
+        else:
+            # Don't allow editing verified properties in any other situation
```

Comment:
May be my understanding of Django. Would this allow a signed in user to maliciously or accidentally unverify events just by not sending "verified" in the API call?

Reply:
Should we just ignore verified fields in this circumstance?

Reply:
Yes, I've done some cleanup, pushing changes in a sec, finishing up the review.

Reply:
Thanks for taking care of this :)